### ISSUE TITLE
[BUG] v0.13.0 image run fail in arm64

### ISSUE THREAD

**Issue by zwwhdls** [juicedata/juicefs-csi-driver#284]
**What happened**:
CSI v0.13.0 runs fail in arm64:

```
standard_init_linux.go:207: exec user process caused "exec format error"
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**

**Environment**:
- JuiceFS CSI Driver version (which image tag did your CSI Driver use): v0.13.0
- Kubernetes version (e.g. `kubectl version`):
- Object storage (cloud provider and region):
- Metadata engine info (version, cloud provider managed or self maintained):
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage):
- Others:

**Comment by zwwhdls** [comment]
fixed by https://github.com/juicedata/juicefs-csi-driver/pull/285

**Comment by stevefan1999-personal** [comment]
> fixed by #285

confirmed
